packages/retail-ui-extensions/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './extension-api';
```

Comment:
How are the details here used? It's odd to me that we do not introduce the notion of "extension points" before introducing APIs extensions have, since in all other extension libraries the extension APIs are just a property of the extension points (https://github.com/Shopify/ui-extensions/blob/main/packages/checkout-ui-extensions/src/extension-points/extension-points.ts#L7-L50 for checkout)